examples/express/index.js

```diff
@@ -0,0 +1,159 @@
+'use strict'
+
+// FOR DEVELOPMENT ONLY: allow for self-signed/invalid certificates universally
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+
+// Use Express.js for routing.
+const express = require('express')
+var session = require('express-session')
+
+const app = express()
+
+app.use(session({
+  secret: 'CHANGE-IT',
+  resave: false,
+  saveUninitialized: true,
+  cookie: {}
+}))
+
+// Import
+const helper = require('openid-client-helper')
+
+// // A ForgeRock configuration example.
+// const issuerMetadata = {
+//   issuer: 'https://default.iam.example.com/am/oauth2',
```

Comment:
Shouldn't this sample include the authz and token endpoints?

Reply:
Probably not, for they can be discovered, while `revocation_endpoint` is not included in AM's configuration document.

Reply:
Then, shouldn't the well-known endpoint be provided?

Reply:
Actually, it is not the .well-known URI itself, but the issuer location, as described in OIDC docs. `openid-client` will discover the .well-known, if it is available. It is all in the main README, but maybe I'll add that here as well.

Reply:
Hmm, well for some reason it didn't work, the way you had it.

Reply:
Works for me, just tried. You un-commented the FR sample and commented out the "abstract" one, correct?

Reply:
At what part does it fail for you?

Reply:
Huh. Strangely, it is now working for me. Who knows